redash/query_runner/athena.py

```diff
@@ -125,18 +125,20 @@ def __get_schema_from_glue(self):
                 region_name=self.configuration['region']
                 )
         schema = {}
-        paginator = client.get_paginator('get_tables')
-
-        for database in client.get_databases()['DatabaseList']:
-            iterator = paginator.paginate(DatabaseName=database['Name'])
-            for table in iterator.search('TableList[]'):
-                table_name = '%s.%s' % (database['Name'], table['Name'])
-                if table_name not in schema:
-                    column = [columns['Name'] for columns in table['StorageDescriptor']['Columns']]
-                    schema[table_name] = {'name': table_name, 'columns': column}
-                    for partition in table['PartitionKeys']:
-                        schema[table_name]['columns'].append(partition['Name'])
-
+        database_paginator = client.get_paginator('get_databases')
+        table_paginator = client.get_paginator('get_tables')
+        databases = (database for databases in database_paginator.paginate()
+                     for database in databases['DatabaseList'])
```

Comment:
Thank you for making the change into two for-loops. But don't we need to remove lines 130-131 now?